meson.build

```diff
@@ -3846,6 +3846,15 @@ public_programs += executable(
         install : true,
         install_dir : rootbindir)
 
+public_programs += executable(
+        'systemd-battery-check',
+        'src/battery-check/battery-check.c',
+        include_directories : includes,
+        link_with : [libshared],
+        dependencies : [userspace, versiondep],
+        install_rpath : rootpkglibdir,
+        install : true)
```

Comment:
This is missing `install_dir : rootlibexecdir`, since the final binary should be installed under `(/usr)/lib/systemd/.